gtsam.h

```diff
@@ -2812,7 +2812,34 @@ virtual class ImuFactor: gtsam::NonlinearFactor {
 };
 
 #include <gtsam/navigation/CombinedImuFactor.h>
+//typedef PreintegratedCombinedMeasurements::Params PreintegrationCombinedParams;
```

Comment:
kill :-)

Reply:
Yes, that's from when I was trying to leave CombinedImuFactor alone!